docs/quickstart/quickstart-non-docker.md

```diff
@@ -15,8 +15,6 @@ This part of the quick start will guide you through the steps to setup a Kafka c
 - KSQL is in developer preview. Do not run KSQL against a production cluster.
 - [Confluent Platform 4.0.0](http://docs.confluent.io/current/installation.html) is installed. This installation includes a Kafka broker, ZooKeeper, Schema Registry, REST Proxy, and Kafka Connect.
   - If you installed Confluent Platform via tar or zip, navigate into the installation directory. The paths and commands used throughout this quick start assume that your are in this installation directory.
-- [Maven](https://maven.apache.org/install.html)
-- [Git](https://git-scm.com/downloads)
```

Comment:
why are we removing Git as a prerequisite?

Reply:
Its not needed to run KSQL using the release tarball.